.circleci/config.yml

```diff
@@ -4,14 +4,22 @@ jobs:
     working_directory: ~/tea-shops
     docker:
       - image: circleci/ruby:2.5.3
+
     steps:
       - checkout
       - run:
           name: bundle install
           command: bundle install
+      - run:
+          name: sequel migration test
+          command: sequel -m db/migrations sqlite://tea-shops_test.sqlite
+
+          environment:
+            APP_ENV: 'test'
```

Comment:
you should move the `environment:...` inside `docker`, then the `APP_ENV` will be available for every run.